libs/designsystem/src/lib/components/modal/footer/modal-footer.component.ts

```diff
@@ -1,32 +1,12 @@
-import { Component, Input, OnInit, ElementRef, OnChanges, SimpleChanges } from '@angular/core';
+import { Component, Input, HostBinding } from '@angular/core';
 
 @Component({
   selector: 'kirby-modal-footer',
   templateUrl: './modal-footer.component.html',
   styleUrls: ['./modal-footer.component.scss'],
 })
-export class ModalFooterComponent implements OnInit, OnChanges {
-  @Input() snapToKeyboard = false;
+export class ModalFooterComponent {
+  @HostBinding('class.snap-to-keyboard') @Input() snapToKeyboard = false;
```

Comment:
Consider readability and code consistency:
```suggestion
  @HostBinding('class.snap-to-keyboard')
  @Input() snapToKeyboard = false;
```